arlas-rest/src/main/java/io/arlas/server/rest/explore/raw/RawRESTService.java

```diff
@@ -99,7 +106,7 @@ public Response getArlasHit(
             throw new NotFoundException("Document " + identifier + " not found.");
         }
 
-        Hit hit = new Hit(collectionReference, source);
+        Hit hit = new Hit(collectionReference, source, (flat!=null && flat));
```

Comment:
If you prefer, there are more readable way to express it (pass a boolean and not a Boolean, or  Boolean.TRUE.equals(flat) )

Reply:
Done :+1: